circus/circusrv.py

```diff
@@ -0,0 +1,115 @@
+import win32serviceutil
```

Comment:
Isn't this the windows counterpart to the `circusd.py` so doesn't it make sense to make circusd systemdependent?